source/content/guides/backups/02-access-backups.md

```diff
@@ -23,7 +23,72 @@ Pantheon backups are stored offsite on Google Cloud Storage instances. We recomm
 
 <Download file="pantheon-backup-to-s3.sh" />
 
-GITHUB-EMBED https://github.com/pantheon-systems/documentation/blob/main/source/scripts/pantheon-backup-to-s3.sh.txt bash GITHUB-EMBED
```

Comment:
@rachelwhitton should we delete these `.txt` files too?

Reply:
No because the [download component](https://github.com/pantheon-systems/documentation/blob/main/src/components/download/index.js#L8) expects it to be there for the button to work above the code snippet (used in cases where we are giving a full bash script for the user to run)
<img width="1046" alt="Screenshot 2024-11-07 at 1 37 59 PM" src="https://github.com/user-attachments/assets/452bb8d2-98f4-47bd-b63c-c3f37a530935">